### ISSUE TITLE
Create a New Site from an Existing Domain

### ISSUE THREAD

**Issue by Gilmoursa** [Automattic/wp-calypso#75494]
### Problem

User created a site with 2 domains instead of creating 2 different sites, which is their goal. The process of creating a new blank site then transferring one of those domains to the new site is cumbersome and unintuitive. The user must:

1. create a new blank site using a free [WordPress.com](<http://WordPress.com>) subdomain (unless they want to purchase a third domain)
2. purchase a plan for their new site
3. return to their original site
4. transfer the domain from their original site to their new site

### Why

We are making sales a priority and users often struggle with the current process outlined above. Making it easier for users to create additional [WordPress.com](<http://WordPress.com>) sites and purchase additional plans from us would help increase sales within WPCOM.

### Proposed Solution

1. User visits: *My Site > Upgrades > Domains*. Then beside the domain, selects *“transfer”*.
2. Within that transfer settings page, the user has a new option available to them (this is the feature I am proposing): ***Transfer this domain to a new blank site.***

Selecting that button brings the user to the new site signup flow with the domain already populated<br>Once the user completes the new site signup, the domain is automatically transferred over for them

**Comment by cuemarie** [comment]
📌 ACTIONS
- Feature request kept
- CC: @Automattic/martech for domains/site creation and purchasing ideas

📌 Message to Author
- Great idea!  This is a user experience I see happen frequently, and I agree, it's cumbersome!  I even have a predef for the situation, because it comes up relatively frequently:

```
The steps you'll need to take in order to create a new WordPress site for this domain (leaving your other site in place as well) are as follows:

1. Select Switch Site > "+ ADD NEW SITE" to begin creating your new site. For now, choose a free ".wordpress.com" domain, and select one of the paid plans, depending on your goals for that site. (Check out a comparison of these plans here: https://wordpress.com/pricing/ )

2. After you purchase a plan and create your new site, you can transfer the domain you already registered from your existing site to your new one, following this guide: https://wordpress.com/support/transfer-a-domain-to-another-wordpress-com-account/#transfer-domain-to-another-site-you-own

Here's a guide that talks through that process too, if that works better for you! https://wordpress.com/support/create-a-blog/#moving-a-custom-domain-to-a-new-site 
```

**Comment by masperber** [comment]
Yes, this is a frequent need for our customers. When they try to create a new site and use a domain that they already own on another site, they'll receive an error message that the domain is already in use. This is confusing, and unnecessarily funnels them into support, when it should be frictionless to create a new site using a domain they already own.

**Comment by zdenys** [comment]
Are there considerations for prioritizing this in light of Google Domains project? cc: @ebinnion 

Another case 6626915-zd-a8c

**Comment by ebinnion** [comment]
> Are there considerations for prioritizing this in light of Google Domains project?

Yes, possibly. Of note, for domain only sites, users can create a site for their domain by visiting the domains management page for that domain, which should then be a pretty seamless flow.

<img width="680" alt="Screenshot 2023-08-14 at 5 24 53 PM" src="https://github.com/Automattic/wp-calypso/assets/1126811/bf32bcaa-740d-4a3c-96fb-1767754636c7">

Was this not an option for this user?

**Comment by zdenys** [comment]
> Was this not an option for this user?

The user didn't have a domain-only site in my case, but a normal site. Thus mine was just a curiosity about the chance of fixing this long-standing issue now that we have an influx of domains and hence the domain-related flows/bugs/issues get an opportunity to be addressed.

**Comment by ebinnion** [comment]
> The user didn't have a domain-only site in my case, but a normal site.

To clarify, the user had multiple domains pointing to a single site and then wanted to use one of the domains for another site? I just want to be very clear that I'm understanding the problem.

**Comment by masperber** [comment]
> To clarify, the user had multiple domains pointing to a single site and then wanted to use one of the domains for another site?

Yes, this happens frequently.

Many users don’t understand the difference between a domain and a website, so they will register a new domain, thinking that they are making a new website. Instead, they add a domain to their existing website.

From this situation, it is not possible to use the domain when you go through the usual website creation flow. You will get an error message that the domain is already in use. Instead, you must create a site with a free domain first, and only after that has been completed, you can move the domain from the old site to the new site.

This process is not intuitive, so it’s usually completed in a support interaction.

**Comment by ebinnion** [comment]
That's helpful. Thank you. Yes, we can consider this as part of our roadmap. Yesterday, @delputnam also suggested that we could add some functionality to allow users to park a domain (make it domain only again) as well. So, I imagine that we could address both points here at the same time.
